package.json

```diff
@@ -27,9 +27,9 @@
   },
   "pnpm": {
     "overrides": {
-      "@babel/core": "^7.23.3",
-      "@babel/traverse": "^7.23.3",
-      "fast-glob": "^3.3.1"
+      "@babel/core": "^7.23.9",
+      "@babel/traverse": "^7.23.9",
+      "fast-glob": "^3.3.2"
```

Comment:
**TODO**

These overrides had been added in #1791 to address vulnerabilities. Check whether the dependencies of `ember-intl` have been updated enough such that these overrides are no longer needed.